### ISSUE TITLE
Empty lines in vim take too long to show

### ISSUE THREAD

**Issue by marusak** [xtermjs/xterm.js#1965]
**This is definitely related to the fact we are using accessibility mode (screenreader support). If I turn off this mode it works just fine**

Opening vim in terminal results in a very long loading time. During this time browser (tested on Chrome and Firefox) becomes unresponsive. Afterwards everything works fine, unless the browser window gets resized which again results in unresponsiveness.

## Details
- Browser and browser version: Firefox 65
- OS version: Fedora 29
- xterm.js version: 3.10.1

### Steps to reproduce
Opening nonexistent or empty file is much slower than opening existing file. Opening empty file takes ~4s, opening file with 1 line takes ~3.8s, opening file with 2 lines takes ~3.5s, 10 lines takes ~2.3s, opening file with more than 20 lines is immediate (out of 40 visible lines). On smaller screen it works proportionally. If I fit on screen only 20 lines and open empty file it is immediate, if I fit 30 lines on screen and open empty file it takes around those 2 seconds.

It seems that every empty line in vim sends something that xterm.js parses for accessibility mode.

**Comment by jerch** [comment]
@Tyriar Im not used to the screenreader stuff with xterm.js - is this a regression with the new buffer?

**Comment by Tyriar** [comment]
@jerch I don't think so, most of the buffer changes happened after 3.10 I think? I suspect it's just about vim doing too many changes and the screen reader support wasn't optimized around the alt buffer/vim.

**Comment by jerch** [comment]
@Tyriar Did some quick debugging with vim and screen reader mode, startup with an empty file takes one second compared to 128 ms when it is disabled (on my machine). Seems this is very costly: https://github.com/xtermjs/xterm.js/blob/fda964b3ea864f83059de146626e0aeb882d1de7/src/AccessibilityManager.ts#L286-L295

especially L290, which triggers HTML reparsing for the browser (basically adds 850 ms to the 128 ms).

Since I have no knowledge about screen reader/NVDA mechanics I cannot help here much (whether it has to be done this way for whatever reason, whether output speed is an issue at all for screen readers).

@marusak Maybe you can disable screen reader by default and provide an option in your app for ppl to set it on when needed?

**Comment by Tyriar** [comment]
We could collect all announced characters and flush them on the next animation frame if that's the problem.

**Comment by jerch** [comment]
@Tyriar Yeah if we can do that, it would cut off most the runtime spent here.
